torch/_inductor/decomposition.py

```diff
@@ -43,6 +43,7 @@
 inductor_decompositions = get_decompositions(
     [
         aten._adaptive_avg_pool2d_backward,
+        aten.alias_copy.default,
```

Comment:
This decomposition can probably go into the default decompositons in `_decompositions/__init__.py` as it is not inductor-specfic.